Code releases 05.20/2da65b72-267d-4d1b-a1b7-c5c4656b86ea.md

```diff
@@ -1,9 +1,7 @@
 This document provides information about setting up the debugging for Spryker in Docker.
 
-[Xdebug](https://xdebug.org) is used to debug setup in Docker. To enable Xdebug, run the command:
-```bash
-docker/sdk {run|start|up} -x
-``` 
+[Xdebug](https://xdebug.org) is used to debug setup in Docker. To enable Xdebug, use a Browser Extension to set a Cookie with XDEBUG_SESSION=PHPSTORM.
```

Comment:
We need to specify the option before 1.15.0 and after.
Also, the text about cookie should be more abstract to cover API. Probably with examples.

@andriitserkovnyi Please, take care of this. Thanks.